### PR TITLE
throw runtime_error when file can not be read or tree does not exist

### PR DESCRIPTION
**Pull request by pbuehler**

### CODE REVIEW

**Comment by pbuehler** [comment]
@ktf Hi Giulio, could you please have a look at this?

File: Framework/Core/include/Framework/DataInputDirector.h [hunk 1]
```diff
@@ -10,6 +10,7 @@
 #ifndef o2_framework_DataInputDirector_H_INCLUDED
 #define o2_framework_DataInputDirector_H_INCLUDED
 
+#include "TGrid.h"
```
Comment (ktf) [inline Framework/Core/include/Framework/DataInputDirector.h:13]: Please do not include TGrid in the headers, unless really needed.

File: Framework/Core/include/Framework/DataInputDirector.h [hunk 2]
```diff
@@ -104,6 +105,7 @@ struct DataInputDirector {
   std::vector<DataInputDescriptor*> mdataInputDescriptors;
 
   bool mdebugmode = false;
+  bool malienSupport = false;
```
Comment (ktf) [inline Framework/Core/include/Framework/DataInputDirector.h:108]: `mAlienSupport`.

**Comment by ktf** [comment]
Thanks!

**Pull request merged**